### DISCUSSION TITLE
Floppy Bridge settings

### DISCUSSION THREAD

**Discussion by ScytheandGenesisGames** [BlitterStudio/amiberry#1011] [Q&A] [answered]
Can someone explain what of the following modes Floppy Bridge does with Amiberry with a Floppy Drive?

FB: Fast
FB: Compatible
FB: Turbo
FB: Accurate

just want to have semi accurate emulation of using Floppy Bridge and Amiberry

**Comment by midwan** [comment, marked as answer]
They represent the different modes that the Floppy Bridge hardware supports.
Faster modes will be faster, but might cause compatibility issues with some scenarios (e.g. they will even pre-load tracks from the floppy disk to have things in memory).

If you're looking for in-depth documentation about how they work, it would be best to ask at Rob Smith's dedicated support channels (e.g. he runs a Discord server)
